marketplace/job/caffe2_resnet50.yaml

```diff
@@ -1,26 +1,29 @@
-protocol_version: v2
-name : caffe2_resnet50
-type : job
-version : 1.0.0
-contributor : Yanli
-description : caffe2, resnet50
-retryCount: 0 
+protocolVersion: 2
+name: caffe2_resnet50
+type: job
+version: 1.0
+contributor: OpenPAI
+description: |
+  # Caffe2 ResNet50 Example
+  This example shows using caffe2 to train ResNet50 with fake data on OpenPAI.
 
-tasks :
-  - role: main
-    dockerimage : caffe2
-    resource: 
-      instances : 1
-      resourcePerInstance: { cpu: 4, memoryMB: 8192, gpu: 1 }
-    minSucceededTaskCount: 1
-    command:
-      - python resnet50_trainer.py --train_data null
-
-prerequisites : 
-  - protocol_version : v2
-    name : caffe2
-    type : dockerimage
-    version : 1.0.0
-    contributor : Yanli
-    description: caffe2
+prerequisites:
+  - protocolVersion: 2
+    name: caffe2_example
+    type: dockerimage
+    version: 1.0
+    contributor : OpenPAI
+    description: |
+      This is an [example caffe2 Docker image on OpenPAI](https://github.com/Microsoft/pai/tree/master/examples/caffe2).
     uri : openpai/pai.example.caffe2
+
+taskRoles:
+  train:
+    instances: 1
+    dockerImage: caffe2_example
+    resourcePerInstance:
+      cpu: 4
+      memoryMB: 8192
+      gpu: 1
+    commands:
+      - python resnet50_trainer.py --train_data null
```

Comment:
Are we changed job description file to v2 format here?

Reply:
It has been changed in #2292, hasn't been merged yet.
Added in a separate directory `protocol-marketplace` and keep the original one.

Reply:
renamed to `marketplace-v2`